### ISSUE TITLE
Cannot render the preview of the artboard

### ISSUE THREAD

**Issue by xorb** [opendesigndev/open-design-sdk#18]
Tried to run basic example: https://opendesign.dev/docs/getting-started. But seems like it is not working as expected.

## Details
-- yarn 1.22.17
-- node v14.18.3
-- @opendesign/sdk : "^0.4.0"

## Output
```
 node --experimental-specifier-resolution=node --loader ts-node/esm.mjs src/index.ts
(node:17248) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
 error | Rendering: stderr> terminate called after throwing an instance of 'OpenglException'
 error | Rendering: stderr>   what():  glfwInit failed
 error | Rendering engine cound not be initialized Error: Rendering process exited with code null
             at ChildProcess.handleClose (/home/xorb/Projects/Diiffer/open-design-import/node_modules/@opendesign/rendering/src/rendering-process.ts:62:16)
             at Object.onceWrapper (events.js:520:26)
             at ChildProcess.emit (events.js:412:35)
             at ChildProcess.emit (domain.js:475:12)
             at maybeClose (internal/child_process.js:1058:16)
             at Process.ChildProcess._handle.onexit (internal/child_process.js:293:5)
(node:17248) UnhandledPromiseRejectionWarning: Error: The rendering engine is not configured
    at DesignFacade.<anonymous> (/home/xorb/Projects/Diiffer/open-design-import/node_modules/@opendesign/sdk/src/design-facade.ts:1341:13)
    at Generator.next (<anonymous>)
    at /home/xorb/Projects/Diiffer/open-design-import/node_modules/@opendesign/sdk/dist/design-facade.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/xorb/Projects/Diiffer/open-design-import/node_modules/@opendesign/sdk/dist/design-facade.js:4:12)
    at DesignFacade.renderArtboardToFile (/home/xorb/Projects/Diiffer/open-design-import/node_modules/@opendesign/sdk/dist/design-facade.js:1084:16)
    at ArtboardFacade.<anonymous> (/home/xorb/Projects/Diiffer/open-design-import/node_modules/@opendesign/sdk/src/artboard-facade.ts:754:30)
    at Generator.next (<anonymous>)
    at /home/xorb/Projects/Diiffer/open-design-import/node_modules/@opendesign/sdk/dist/artboard-facade.js:8:71
    at new Promise (<anonymous>)
(node:17248) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:17248) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 14.03s.
```

**Comment by petrbrzek** [comment]
Hi @xorb, thanks for the issue. We are able to reproduce the bug and are working on a fix.

**Comment by janroztocil** [comment]
There indeed is a misconfiguration of the package when using OpenGL acceleration on Linux. We're about to fix it in the next update. Nevertheless, in the meantime as a workaround you can fallback to Off-screen Mesa Software render. Simply put the following lines into your project local `.npmrc` file:

```
monroe_cli_install_deps=true
monroe_cli_gl_provider=osmesa
```

... or alternatively (if you don't run your project via npm/yarn) set the environment variables:

```
export MONROE_CLI_INSTALL_DEPS=true                                                                                                                    
export MONROE_CLI_GL_PROVIDER=osmesa
```

**Comment by janroztocil** [comment]
Fixed in version `0.4.2`.

Now on desktop Linux the correct dependencies are fetched. Note that if the installer detects a running X Window system it assumes OpenGL is set up so that internal rendering module could take advantage of hardware acceleration.

If for whatever reason the software rendering is desired then the configuration suggested in the post above is the way to set it up.
